src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/Server/SqlSer.cs

```diff
@@ -151,72 +30,19 @@ internal static SqlUserDefinedTypeAttribute GetUdtAttribute(Type t)
             return udtAttr;
         }
 
-        // Create a new serializer for the given type.
-        private static Serializer GetNewSerializer(Type t)
-        {
-            SqlUserDefinedTypeAttribute udtAttr = GetUdtAttribute(t);
-            Format k = GetFormat(t);
-
-            switch (k)
-            {
-                case Format.Native:
-                    return new NormalizedSerializer(t);
-                case Format.UserDefined:
-                    return new BinarySerializeSerializer(t);
-                case Format.Unknown: // should never happen, but fall through
-                default:
-                    throw ADP.InvalidUserDefinedTypeSerializationFormat(k);
-            }
-        }
-    }
-
-    // The base serializer class.
-    internal abstract class Serializer
-    {
-        protected Type _type;
-
-        public abstract object Deserialize(Stream s);
-        public abstract void Serialize(Stream s, object o);
-
-        protected Serializer(Type t) => _type = t;
-    }
-
-    internal sealed class NormalizedSerializer : Serializer
-    {
-        private BinaryOrderedUdtNormalizer _normalizer;
-        private bool _isFixedSize;
-        private int _maxSize;
-
-        internal NormalizedSerializer(Type t) : base(t)
-        {
-            SqlUserDefinedTypeAttribute udtAttr = SerializationHelperSql9.GetUdtAttribute(t);
-            _normalizer = new BinaryOrderedUdtNormalizer(t, true);
-            _isFixedSize = udtAttr.IsFixedLength;
-            _maxSize = _normalizer.Size;
-        }
-
-        public override void Serialize(Stream s, object o) => _normalizer.NormalizeTopObject(o, s);
-
-        public override object Deserialize(Stream s) => _normalizer.DeNormalizeTopObject(_type, s);
+      
     }
 
-    internal sealed class BinarySerializeSerializer : Serializer
+    internal sealed partial class BinarySerializeSerializer : Serializer
     {
-        internal BinarySerializeSerializer(Type t) : base(t)
-        {
-        }
-
-        public override void Serialize(Stream s, object o)
+        private void SerializeFrameworkSpecific(Stream s, object o)
         {
             BinaryWriter w = new BinaryWriter(s);
             ((IBinarySerialize)o).Write(w);
         }
 
-        // Prevent inlining so that reflection calls are not moved
-        // to a caller that may be in a different assembly that may
-        // have a different grant set.
-        [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
This attribute and it's explanation seem important to keep.

Reply:
@Wraith2 This method, attribute and comment has been moved to SqlSer.Common.cs

Reply:
@Wraith2  This methods, attribute and comment has been moved to SqlSer.Common.cs

Reply:
Ok. Diff comprehension failure on my part then.

Reply:
Understood.